DirectProgramming/C++SYCL_FPGA/README.md

```diff
@@ -94,14 +94,13 @@ flowchart LR
 | [loop_unroll](Tutorials/Features/loop_unroll)                                                                                 | [Tutorials/Features](Tutorials/Features)                                                 | Basics of loop unrolling. <br> How to unroll loops in your program <br> Determining the optimal unroll factor for your program
 | [max_interleaving](Tutorials/Features/max_interleaving)                                                                       | [Tutorials/Features](Tutorials/Features)                                                 | The basic usage of the `max_interleaving` attribute <br> How the `max_interleaving` attribute affects loop resource use <br> How to apply the `max_interleaving` attribute to loops in your program
 | [memory_attributes](Tutorials/Features/memory_attributes)                                                                     | [Tutorials/Features](Tutorials/Features)                                                 | The basic concepts of on-chip memory attributes <br> How to apply memory attributes in your program <br> How to confirm that the memory attributes were respected by the compiler <br> A case study of the type of performance/area trade-offs enabled by memory attributes
-| [mmhost](Tutorials/Features/hls_flow_interfaces/mmhost)                                                                   | [Tutorials/Features/hls_flow_interfaces](Tutorials/Features/hls_flow_interfaces) | Basics of declaring Avalon memory-mapped host data interfaces for IP components
+| [mmhost](Tutorials/Features/hls_flow_interfaces/mmhost)                                                                       | [Tutorials/Features/hls_flow_interfaces](Tutorials/Features/hls_flow_interfaces) | Basics of declaring Avalon memory-mapped host data interfaces for IP components

```

Comment:
and for all of the hls_flow_interfaces samples?

Reply:
adding spaces to align the table divider line "|"